Wireframe/css/style.css

```diff
@@ -0,0 +1,163 @@
+/* Reset default margin for body element */
+body {
+  margin: 0;
+  background-color: #b4b4b4;
```

Comment:
Instead of setting all the hex colours throughout the file, how could you store these values and reuse them?

Reply:
after some research, I saw some of the CYF projects that some of the pages in the CSS file there is a root custom properties, which I had no idea why, my first suggestion was maybe it was related to javascript coding. 
but now I understand why to restore those values we can use a root custom properties like this 
:root {
    --bg-color : #b4b4b4;
}

/* implementing */
body {
    background-color: var(--bg-color);
}

now I will make these adjustments :pray: